verify.php

```diff
@@ -8,11 +8,12 @@
 if($validateStats === false){
     header('location: char_creation.php?error=Stats must be below 20');
 }
-
-$charname = filter_var($_POST['charname'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
-$class = filter_var($_POST['class'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
+$filter = sanitizePost();
+$charname = $filter[0];
+$class = $filter[1];
+$image = $filter[2];
```

Comment:
if you change sanitizePost as mentioned, this bit of code will read a lot better